portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -6,14 +6,45 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        response.setContentType("text/html;");
-        response.getWriter().println("<h1>Hello Cesar!</h1>");
+
+
+   
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    ArrayList<String> cars = new ArrayList<String>();
+    cars.add("Volvo");
+    cars.add("BMW");
+    cars.add("Ford");
+ 
+
+    String json = convertToJson(cars);
+
+    // Send the JSON as the response
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+
   }
-}
+
+
+    private String convertToJson(ArrayList<String> cars) {
```

Comment:
The Gson library should be able to do this kind of conversion for you I think, though you'd have to organize things differently if you wanted to use the names "firstCar", etc.. instead of using a list/array.